packages/blade/src/components/Amount/AmountValue.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ReactElement } from 'react';
+
+import type { FontSize } from '../../tokens/global/typography';
+import type { BaseTextProps } from '../Typography/BaseText/types';
+import { amountTextSizes, prefixSuffixTextSizes } from './amountTokens';
+import type { AmountProps } from './Amount';
+import { suffixTypes } from './Amount';
+import BaseBox from '~components/Box/BaseBox';
+import { BaseText } from '~components/Typography/BaseText';
+
+const getAffixFontWeight = (isAffixSubtle: true | false): 'regular' | 'bold' => {
+  if (isAffixSubtle) return 'regular';
+  return 'bold';
+};
+
+const getMainValuetWeight = (size: NonNullable<AmountProps['size']>): 'regular' | 'bold' => {
+  return size.includes('bold') || size.includes('title') ? 'bold' : 'regular';
```

Comment:
use startsWith for more accuracy?
```suggestion
  return size.includes('bold') || size.startsWith('title') ? 'bold' : 'regular';
```